packages/caliper-ethereum/lib/ethereum-connector.js

```diff
@@ -302,7 +302,7 @@ class EthereumConnector extends ConnectorBase {
 
         const onFailure = (err) => {
             status.SetStatusFail();
-            logger.error('Failed private tx on ' + request.contract + ' calling method ' + request.verb + ' private nonce ' + 0);
+            logger.error(`Failed private tx on ${request.contract}; calling method: ${request.verb}; private nonce: ${0}`);
```

Comment:
not sure why the original log entry had `+ 0` but it can't be replaced with `${0}` here. I think it might be the value of `sender.nonce` if set and literal `0` otherwise

Reply:
Is it ``privacy.nonce`` or sender.nonce``?
Shall I add a condition to check if ``sender.nonce`` is set ? If not then I'll set it to 0 and then will replace the ``0`` above with the value of ``sender.nonce``.
Is it fine to do it this way?

Reply:
My recommendation would be to not change the message around the nonce. If you change it to sender.nonce then you will need to write unit tests and prove that it works. So suggest you just do
```
logger.error(`Failed private tx on ${request.contract}; calling method: ${request.verb}; private nonce: ` + 0);
```

Reply:
Ok, I'll change it.